server/tests/graphql.rs

```diff
@@ -0,0 +1,41 @@
+use crate::common::{
+    auth::get_token,
+    fixture::{new_id, LLDAPFixture, User},
+    graphql::{post, ListUsers},
+};
+use reqwest::blocking::ClientBuilder;
+use serial_test::file_serial;
+use std::collections::HashSet;
+mod common;
+
+#[test]
+#[file_serial]
+fn list_users() {
+    let mut fixture = LLDAPFixture::new();
```

Comment:
They are talking to the same db. The idea is that anything created by the fixture is automatically cleaned up, and each t est sets up it''s own users and groups to avoid any dependence on the state before

Reply:
We'll have to make double-sure for tests that check the entire list of users, for instance.

Reply:
Yeah, i think we want to avoid things like "get a list of all users and expect size to be exactly two since we added 2 users in this test"

Reply:
I agree with the precaution, but I think we'll need some tests like that.
Wondering: can't we just create databases (not DB servers) on the fly? For SQlite it's different files, but for postgres/mysql you can create different DBs.

Reply:
Note: if you want to merge like that, it's fine with me, we can have another pass later on.